hub/policy/src/test/java/uk/gov/ida/hub/policy/builder/state/IdpSelectedStateBuilder.java

```diff
@@ -27,6 +27,7 @@
     private SessionId sessionId = SessionIdBuilder.aSessionId().build();
     private List<String> availableIdentityProviders = List.of("idp-a", "idp-b", "idp-c");
     private boolean transactionSupportsEidas = false;
+    private String abTestVariant;
```

Comment:
No longer needed

Reply:
👍 It's all looking good to me just a few things above.

Reply:
This is used in the file.